core/src/main/java/org/springframework/security/access/intercept/AbstractSecurityInterceptor.java

```diff
@@ -115,6 +118,8 @@
 
 	private AccessDecisionManager accessDecisionManager;
```

Comment:
It might make sense to consider introducing a new filter that will use an `AuthorizationManager` similar to `AuthorizationWebFilter`.

Reply:
Agreed that it's worth considering. It doesn't change the fact that `AbstractSecurityInterceptor` needs to stop using `AccessDecisionManager`, but it would probably be worth it to file a ticket and start that conversation.